cms/src/components/forms/LChecklist.spec.ts

```diff
@@ -0,0 +1,79 @@
+//@ts-nocheck --> To prevent wrapper.vm.givenOptions typing error as it is valid code - block level ignore doesn't work so this is the best solution for that.
```

Comment:
A bit dangerous, but as it is a test file we can live with it

Reply:
I also don't like this approach but typescript is giving type issues for valid code and I used that code a lot of times so without this the file will be polluted with ```@ts-ignore```. If someone needs to add a test they can just remove this line temporarily, and then re-add it afterwards. From my research I discovered that  a lot of developers also struggle with this during testing.